Source/SourceKittenFramework/String+SourceKitten.swift

```diff
@@ -393,30 +394,29 @@ extension NSString {
 
 extension String {
     internal var isFile: Bool {
-        return NSFileManager.defaultManager().fileExistsAtPath(self)
+        return FileManager.default.fileExists(atPath: self)
     }
 
     /// Returns the `#pragma mark`s in the string.
     /// Just the content; no leading dashes or leading `#pragma mark`.
-    public func pragmaMarks(filename: String, excludeRanges: [NSRange], limitRange: NSRange?) -> [SourceDeclaration] {
+    public func pragmaMarks(_ filename: String, excludeRanges: [NSRange], limitRange: NSRange?) -> [SourceDeclaration] {
```

Comment:
Inside of this `extension String`, most of function's first parameter label is not compatible with `master` excepting `pragmaMarks()`.